Vitis_Platform_Creation/Design_Tutorials/03_Edge_VCK190/step2.md

```diff
@@ -14,133 +14,388 @@
-   - PetaLinux 2021.2 requires GCC version >= 6.0. If your GCC version doesn't meet this requirement, please enable **Enable buildtools extended** from **petalinux-config → Yocto settings**, which uses the pre-compiled gcc binaries from the PetaLinux tool. For more information, please refer to UG1144.
+### Create the device tree file
 
+Utilize XSCT tool to execute one command to generate device tree files:
+
+   ```bash
+   cd custom_vitis_platform
+   xsct 
+   ```
+   Then execute ` createdts ` command in XSCT console like below:
 
+   ```bash
+   createdts -hw ../custom_hardware_platform/custom_hardware_platform.xsa -zocl  -out .  \
+   -platform-name mydevice  -git-branch xlnx_rel_v2022.1  -compile
+   ```
+   The `createdts` command has the following main input options:
 
+   -  `-name`: Platform name
+   -  `-hw`: Hardware XSA file with path
+   -  `-git-branch`: device tree branch
+   -  `-zocl`: enable the zocl driver support
+   -  `-compile`: specify the option to compile the device tree
 
-### Customize Root File System, Kernel, Device Tree and U-boot
+   Notice below information would show in XSCT console. Please ignore the warning and that means you succeed to get system.dtb file which is located in <mydevice/psu_cortexa53_0/device_tree_domain/bsp>.
 
-1. Add XRT to rootfs packages
-   
-   - Run `petalinux-config -c rootfs`. 
-   - In the configuration window, use `/` key to open search function. 
-   - Type in `xrt` and press Enter. 
-   - In the search result page, press the number of `Symbol: xrt` result, for example **1**. 
-   - In the page of XRT configuration page, go to the line of `xrt` and press space to enable it.
-   - Select Exit twice to return to the main configuration page.
+   ```bash
+   pl.dtsi:9.21-71.4: Warning (unit_address_vs_reg): /amba_pl@0: node has a unit name, but no reg property
+   pl.dtsi:52.26-56.5: Warning (simple_bus_reg): /amba_pl@0/misc_clk_0: missing or empty reg/ranges property
+   pl.dtsi:57.15-65.5: Warning (simple_bus_reg): /amba_pl@0/zyxclmm_drm: missing or empty reg/ranges property
+   pl.dtsi:66.42-70.5: Warning (simple_bus_reg): /amba_pl@0/aie_core_ref_clk_0: missing or empty reg/ranges property
+   pl.dtsi:9.21-71.4: Warning (unique_unit_address): /amba_pl@0: duplicate unit-address (also used in node /memory@0)
+   ```
 
-   ![missing image](./images/step2/petalinux_enable_xrt.png)
+   > Note: Createdts is a command executing in XSCT console to generate device files. This command needs several inputs to generate the device tree files. Regarding the meaning of every option you can execute help command to check the details. Besides XSCT is a Console tool of Vitis. You can start it by typing `xsct` in Linux terminal to start it according to above operation. Or, you can select menu **Xilinx > XSCT Console ** to start the XSCT tool after you launch Vitis.
```

Comment:
Please use all small character for `createdts` because it's a command. If necessary, use Markdown single quote `.